osbs/utils.py

```diff
@@ -316,12 +318,14 @@ def make_name_from_git(repo, branch, limit=53, separator='-'):
 
     repo = git_repo_humanish_part_from_uri(repo)
     branch = branch or 'unknown'
+    hash_str = b64encode(sha256(repo + branch).hexdigest())[:hash_size]
```

Comment:
Wait, I see what you mean. We should be applying b64encode to the actual value of the hash, not to its ASCII representation.